core/src/main/java/hudson/slaves/CloudRetentionStrategy.java

```diff
@@ -47,6 +47,20 @@ public CloudRetentionStrategy(int idleMinutes) {
         this.idleMinutes = idleMinutes;
     }
 
+    /**
+     * @since FIXME
+     */
+    public int getIdleMinutes() {
+        return idleMinutes;
+    }
+
+    /**
+     * @since FIXME
+     */
+    public void setIdleMinutes(int idleMinutes) {
```

Comment:
Use case for example broken deserialisation, that part of other xstream story but you must allow using normal objects and configure them as workaround.
![screen shot 2016-09-20 at 21 27 41](https://cloud.githubusercontent.com/assets/231611/18690306/20e47ffa-7f96-11e6-9453-1141663c7d43.png)